PaypalServerSdk.Standard/Models/ShipmentCarrier.cs

```diff
@@ -1592,10 +1592,10 @@ public enum ShipmentCarrier
         YURTICIKARGO,
 
         /// <summary>
-        /// CNPAYPALPACKAGE.
+        /// CNPaypalPACKAGE.
```

Comment:
This seems like it's breaking convention. I wouldn't expect the word ignore to apply to enums I guess.

Reply:
Oof yeah. I'll bring this one to APIMatic